optimum/amd/ryzenai/modeling.py

```diff
@@ -623,6 +624,52 @@ def forward(self, pixel_values):
 
         return ImageClassifierOutput(logits=next(iter(outputs.values())))
 
+    @classmethod
+    def _export(
+        cls,
+        model_id: str,
+        config: "PretrainedConfig" = None,
+        use_auth_token: Optional[Union[bool, str]] = None,
+        revision: Optional[str] = None,
+        force_download: bool = False,
+        cache_dir: Optional[str] = None,
+        subfolder: str = "",
+        # use_auth_token=use_auth_token,
+        local_files_only: bool = False,
+        trust_remote_code: bool = False,
+        vaip_config: Optional[str] = None,
+        # vaip_config=vaip_config,
+        provider: str = "CPUExecutionProvider",
+        session_options: Optional[ort.SessionOptions] = None,
+        provider_options: Optional[Dict[str, Any]] = None,
+        use_io_binding: Optional[bool] = None,
+        task: Optional[str] = None,
+    ) -> "RyzenAIModel":
+
+        if model_id.startswith("timm"):
```

Comment:
Also, here you could remove this restriction, it should ideally work for all the image-classification model from hub and local path.

Reply:
Yes, you are right